### ISSUE TITLE
you got it working for jetson tx2 ?

### ISSUE THREAD

**Issue by yogeshwar607** [curtismuntz/jetson-wifi-connect#1]

**Comment by curtismuntz** [comment]
Yes, I was able to get this to work on the TX2 if you use one of the supported devices: https://github.com/balena-io/wifi-connect#supported-boards--dongles

And pretty much every one of the wireless devices listed here works as well: https://elinux.org/Jetson_Nano

In my testing, it will not work out of the box using the built-in Broadcom WiFi chipset on the TX1 and TX2.
